Screen/ListScreen.tsx

```diff
@@ -1,26 +1,16 @@
 import React from 'react';
 
-import {View, Text, StyleSheet} from 'react-native';
+import {View} from 'react-native';
 import Weather from '../Components/weather';
-//import Weather from './Components/weather';
 
 class ListScreen extends React.Component {
   render() {
     return (
-      <View style={styles.screenContainer}>
-        <Text>List Screen</Text>
+      <View>
```

Comment:
Do you really need this view ? 
Maybe the TabScreen "Weather" should be your Weather component

Right now you have Weather TabScreen -> View with Weather inside -> Weather list

I think you can do Weather TabScreen -> Weather list